pmm-app/src/shared/components/helpers/api.ts

```diff
@@ -55,6 +58,22 @@ export class ApiRequest {
         // Notify.error(e.message);
       });
   }
+
+  async postCancelable<T, B>(path: string, body: B, disableNotifications = false): Promise<void | T> {
```

Comment:
You probably didn't pull my branch before creating this one, because I dumped this solution last week (it's not canceling requests now but rather is blocking the table to not let the user select a query while the other is loading). It was merged to PMM-2.0 earlier today. Anyway we just need to remove all changes on the api file and replace postCancelable back to post, or remove my commits from this branch. And after that rebase with PMM-2.0 to recheck.